tests/src/Kernel/Entity/OgRoleTest.php

```diff
@@ -243,6 +243,47 @@ public function testRoleCreate() {
     }
   }
 
+  /**
+   * Tests searching roles by a given permission list.
+   */
+  public function testLoadRoleByPermissions() {
+    /** @var \Drupal\og\OgRoleManagerInterface $role_manager */
+    $role_manager = \Drupal::service('og.role_manager');
+    OgRole::create()
+      ->setName(mb_strtolower($this->randomMachineName()))
+      ->setLabel($this->randomString())
+      ->setGroupType('node')
+      ->setGroupBundle('group')
+      ->grantPermission('administer group')
+      ->grantPermission('access content')
+      ->save();
+
+    OgRole::create()
+      ->setName(mb_strtolower($this->randomMachineName()))
+      ->setLabel($this->randomString())
+      ->setGroupType('node')
+      ->setGroupBundle('group')
+      ->grantPermission('access content')
+      ->save();
+
+    $og_role3 = OgRole::create();
+    $og_role3->setName(mb_strtolower($this->randomMachineName()))
+      ->setLabel($this->randomString())
+      ->setGroupType('entity_test')
+      ->setGroupBundle('group')
+      ->grantPermission('administer group')
+      ->save();
+
+    $roles = $role_manager->getRolesByPermissions(['access content']);
+    $this->assertCount(2, $roles);
```

Comment:
Can we also test retrieval of roles with multiple permissions, and the optional parameters `$entity_type_id` and `$bundle`?